app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitViewFragment.java

```diff
@@ -52,6 +54,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         if(getArguments() != null) {
             _habit = (Habit) getArguments().getSerializable("habit");
             _userId = (String) getArguments().getSerializable("userId");
+            // Whether the habit is owned by the current user
+            try {
```

Comment:
Maybe there is a better solution than using a try catch? Not the end of the world tho.